prometheus/config.go

```diff
@@ -53,6 +55,31 @@ type Configuration struct {
 	// on the specified listen address or registering a metric with the
 	// Prometheus. By default the registerer will panic.
 	OnError string `yaml:"onError"`
+
+	// Registry specifies what registry to use and with which default
+	// collectors to use, by default the default Prometheus registry is used
+	// with whichever collectors have already been registered with the
+	// registry.
+	Registry *RegistryConfiguration `yaml:"registry"`
+}
+
+// RegistryConfiguration specifies what registry to use and with which default
+// collectors to use, by default the default Prometheus registry is used
+// with whichever collectors have already been registered with the
+// registry.
+type RegistryConfiguration struct {
+	Default          bool `yaml:"default"`
+	GoCollector      bool `yaml:"goCollector"`
+	ProcessCollector bool `yaml:"processCollector"`
+}
+
+// DefaultRegistryConfiguration returns the default registry configuration.
+// By default the default Prometheus registry is used and no collectors
```

Comment:
This isn’t completely accurate because the default registry registers the go/process collectors by default

Reply:
Sure, but I don't actually register further collectors - which you could do if you enable them.

I'll make it more clear though, as yes the default registry comes with some collectors.